LCARS/Scripts/admin.js

```diff
@@ -44,19 +44,12 @@ function getScreen(id) {
         $("#id").val(data.Id);
         $("#name").val(data.Name);
 
-        var boardRow = "";
-
-        if (data.Boards.length > 0) {
-            $.each(data.Boards, function(key, board) {
-                boardRow += "<tr data-index=\"" + key + "\" class=\"board\">" +
-                    "<td class=\"left\"><div class=\"apricot\">" + board.Category + "</div></td>" +
-                    "<td class=\"middle\">" + board.Argument + "</td>" +
-                    "<td class=\"right\"><div class=\"apricot\">&nbsp;</div></td>" +
-                    "</tr>";
-            });
-        }
+        if (data.Boards.length) {
+            var source = $("#boardsTemplate").html();
+            var template = Handlebars.compile(source);
 
-        $("#boards").html(boardRow);
+            $("#boards").html(template(data));
```

Comment:
I had this outside the "if" before, because if there are no boards I need it to render blank into the div/table. Otherwise the last item when removed doesn't disappear.